### DISCUSSION TITLE
Cloning Router Configuration/Settings to Another Identical Router

### DISCUSSION THREAD

**Discussion by foxrid3r** [dangowrt/owrt-ubi-installer#210] [Q&A]
I have about 10 Linksys E8450 routers that I use at work to connect to various machines on our floor. One router gets connected to each machine for temporary wireless access to the machine during the debug phase of the project. These machines eventually get shipped to our customers and we remove the router before shipping so that it can be used on another machine. Typically, the wireless network name and IP address of the router change from machine to machine--all other settings usually remain the same. I'm looking for some advice on how to properly clone a router to another router so that I don't have to go through the process of installing packages every time a router gets hard reset.

**Comment by mrkiko** [comment]
Hello!!

your question seems to be more general and not specific to this device.
I would pose it in the OpenWrt forum:
https://forum.openwrt.org/

or in the mailing list.

**Comment by foxrid3r** [comment]
Thank you for correcting me. I read in the Wiki that the installer should only be executed once on this device and wanted to confirm that flashing new images can still be done. If this is that case, then I think I can figure out how to create an image and use it on this device with all the documentation out there for Image Builder.
